rust/examples/Cargo.toml

```diff
@@ -27,3 +27,7 @@ phoenix-v1 = { workspace = true }
 phoenix-sdk = { path = "../phoenix-sdk" }
 generic-token-faucet = "0.1.2"
 phoenix-seat-manager = { workspace = true }
+solana-cli-config = "1.14.7"
+reqwest = "0.11.14"
+serde_json = "1.0.94"
+bincode = "1.3.3"
```

Comment:
None of the other packages in the workspace use these so I thought it might be more direct to have them here. I'm not sure what best practice for this is though